exporting/tests/test_exporting_engine.c

```diff
@@ -468,7 +468,7 @@ static void test_format_dimension_stored_json_plaintext(void **state)
         "{\"prefix\":\"netdata\",\"hostname\":\"test-host\",\"host_tags\":\"TAG1=VALUE1 TAG2=VALUE2\","
         "\"chart_id\":\"chart_id\",\"chart_name\":\"chart_name\",\"chart_family\":\"(null)\"," \
         "\"chart_context\": \"(null)\",\"chart_type\":\"(null)\",\"units\": \"(null)\",\"id\":\"dimension_id\","
-        "\"name\":\"dimension_name\",\"value\":690565856.0000000,\"timestamp\": 15052}\n");
+        "\"name\":\"dimension_name\",\"value\":673788640.0000000,\"timestamp\": 15052}\n");
```

Comment:
Why are you changing the constant here?

Reply:
Because, in the original version of my PR, I changed the representation of storage_number to consider the `SN_EXISTS_100` bit flag during unpacking. The updated PR contains only the necessary changes to free the SN_EXISTS bit. I'll post a separate PR to fix the SN_EXISTS_100 issue, because it make review (and my understanding) more complex than what it has to be.

Reply:
Thank you for the explanation. I will review this PR again, when you finish the work and remove the draft.